app/controllers/problems_controller.rb

```diff
@@ -61,7 +61,7 @@ def show
   end
 
   def create_issue
-    body = render_to_string "issue_trackers/issue", layout: false, formats: [:txt]
+    body = render_to_string "issue_trackers/issue", layout: false, formats: [:md]
```

Comment:
Can you explain that a bit more? Right now this is the action that we execute when you click 'create issue' from an issue page. Are you imagining a totally different flow?